src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -207,6 +207,15 @@ Plugin::Plugin()
     // parse again env_variables after backend is initialized to get backend proprieties
     _globalConfig.parseEnvVars();
 
+    // initialize properties which have device-tied default values in global config
+    // *only if there is a driver available
+    if (_metrics->GetAvailableDevicesNames().size() > 0) {
```

Comment:
is everyone happy with this method of "guessing that we are offline"? I am not completely convinced that there is no better approach...
An other solution would be to initialize and store at the first use, but this would mean we execute and reinitialize it each time.. which i also don't particularly like